xcodeproj/internal/runner.template.sh

```diff
@@ -68,12 +68,21 @@ touch /tmp/rules_xcodeproj/WORKSPACE
 echo 'exports_files(["top_level_cache_buster"])' > /tmp/rules_xcodeproj/BUILD
 date +%s > "/tmp/rules_xcodeproj/top_level_cache_buster"
 
+bazel_cmd=(
+  env -i
+  HOME="$HOME"
+  PATH="/opt/homebrew/bin:/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin"
```

Comment:
```suggestion
  PATH="/usr/bin:/bin:/usr/sbin:/sbin:/opt/homebrew/bin:/usr/local/bin"
```

I would recommend this so system binaries get preferred over brew

Reply:
:+1: I'll do that in the next PR.